ui/src/containers/BeamlineSetupContainer.jsx

```diff
@@ -280,17 +279,21 @@ class BeamlineSetupContainer extends React.Component {
               />
             </Nav.Item>
           </Nav>
-          <Nav className="me-3">
+          <Nav className="">
             <Nav.Item>
-              <SampleChangerSwitch
+              <InOutSwitch
+                onText="Power On"
+                offText="Power Off"
+                onValue="PowerOn"
+                offValue="PowerOff"
                 labelText="Sample Changer"
-                data={this.props.sampleChanger.state}
+                data={{ value: this.props.sampleChanger.state }}
```

Comment:
Makes me wonder whether the prop should be `value` directly.

Reply:
Agreed